Sync/Synchronizers/ClientsSynchronizer.swift

```diff
@@ -111,8 +111,14 @@ open class ClientsSynchronizer: TimestampedSingleCollectionSynchronizer, Synchro
     }
 
     var localClients: RemoteClientsAndTabs?
+    // Indicates whether the local client record has been updated to used the
+    // FxA Device ID rather than the native client GUID
     var clientGuidIsMigrated: Bool = false
 
+    // Indicates whether the local client records include a record with an ID
+    // matching the FxA Device ID
+    var ourClientRecordExists: Bool = false
```

Comment:
actually I think this is wrong - once it sets set to true it will never transition back. I think `ourClientRecordExists` can just be a local variable rather than on an instance.

Reply:
Whoops! Thanks for catching that, pushed the fix.